src/block-components/progress-bar/edit.js

```diff
@@ -0,0 +1,137 @@
+/**
+ * External dependencies
+ */
+import { unescape, debounce } from 'lodash'
+import { i18n } from 'stackable'
+import {
+	PanelAdvancedSettings,
+	InspectorStyleControls,
+	AdvancedToggleControl,
+	AdvancedRangeControl,
+	AdvancedTextControl,
+	ColorPaletteControl,
+} from '~stackable/components'
+import { Typography } from '~stackable/block-components'
+import {
+	useBlockAttributesContext, useBlockSetAttributesContext, useAttributeEditHandlers,
+} from '~stackable/hooks'
+import {
+	DEFAULT_PERCENT, DEFAULT_THICKNESS, DEFAULT_SIZE,
+} from './attributes'
+
+/**
+ * WordPress dependencies
+ */
+import {
+	Fragment, useState, useEffect, useCallback,
+} from '@wordpress/element'
+import { escapeHTML } from '@wordpress/escape-html'
+import { __ } from '@wordpress/i18n'
+
+export const Edit = ( { attrNameTemplate } ) => {
+	const setAttributes = useBlockSetAttributesContext()
+	const {
+		progressAnimate,
+		progressRounded,
+	} = useBlockAttributesContext()
+
+	const {
+		getAttribute,
+		updateAttribute,
+	} = useAttributeEditHandlers( attrNameTemplate )
+
+	const progressAriaValueText = getAttribute( 'progressAriaValueText' )
+	const [ valueText, setValueText ] = useState( progressAriaValueText )
+
+	const contentChangeHandler = useCallback( debounce( newValueText => {
+		updateAttribute( 'progressAriaValueText', newValueText )
+	}, 750 ), [] )
+
+	useEffect( () => {
+		contentChangeHandler( valueText )
+	}, [ valueText ] )
+
+	return (
+		<Fragment>
+			<InspectorStyleControls>
+				<PanelAdvancedSettings
+					title={ __( 'Progress Bar', i18n ) }
+					initialOpen
+				>
+					<AdvancedRangeControl
+						label={ __( 'Percentage', i18n ) }
+						default={ DEFAULT_PERCENT }
+						attribute="progressPercent"
+						min="0"
+						sliderMax="100"
+						step="1"
+						placeholder=""
+						isDynamic
+					/>
+					<AdvancedRangeControl
+						label={ __( 'Size', i18n ) }
+						attribute="progressSize"
+						step="1"
```

Comment:
I think we can tweak this for more common-scenarios:
```suggestion
						min="0"
						sliderMin="50"
						sliderMax="300"
						step="1"
```